armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -200,6 +203,10 @@ EOF
 # retain journal logs between reboots 
 ln -sf /mnt/ssd/system/journal/ /var/log/journal
 
+# SYSTEM CONFIGURATION ---------------------------------------------------------
+SYSCONFIG_PATH="/opt/shift/sysconfig/"
+mkdir -p $SYSCONFIG_PATH
+echo "BITCOIN_NETWORK=testnet" > ${SYSCONFIG_PATH}BITCOIN_NETWORK
```

Comment:
(Optional) If we omitted the trailing `/` in `SYSCONFIG_PATH` on L207, we'd get a more natural expression for the full path here:
```
echo "BITCOIN_NETWORK=testnet" > ${SYSCONFIG_PATH}/BITCOIN_NETWORK
```